custom_components/opnsense/sensor.py

```
@@ -41,322 +42,370 @@
+                native_unit_of_measurement=None,
+                icon="mdi:check-network-outline",
+                state_class=None,
+                # entity_category=ENTITY_CATEGORY_DIAGNOSTIC,
+            ),
+            enabled_default=True,
+        )
+        entities.append(entity)
+    return entities
+
+
+async def _compile_interface_sensors(
```

Ruff

`_compile_interface_sensors` is too complex (14 > 10)

(C901)